homeassistant/components/habitica/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
I am missing tests where we try to setup the same device twice and see it abort

Reply:
I'd rather postpone some of these changes until after the library switch, I don't want to do everything twice